oap-server/server-library/library-util/pom.xml

```diff
@@ -43,7 +47,7 @@
         <dependency>
             <groupId>com.github.stefanbirkner</groupId>
             <artifactId>system-rules</artifactId>
-            <version>1.18.0</version>
+            <version>${ststem-rules.version}</version>
```

Comment:
Is this dependency still necessary?

Reply:
Are you still have some test cases in oap-server side?

Reply:
This dependency is necessary for the test case in the library-util module of oap-server side.